packages/client/src/components/NavBar/index.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react";
+import IconButton from '@material-ui/core/IconButton';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Grid from '@material-ui/core/Grid';
+import { Link } from 'react-router-dom';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import PropTypes from 'prop-types';
+import Modal from '../Modal';
+import Wizard from '../Wizard';
+import Button from '../Button';
+import styled from 'styled-components';
+import { COLORS } from '../../constants';
+
+
+const Wiz_content = ['page1', <Button>oh boi</Button>, 'page3']
+
+const TextThemes = {
+  fontSize: {
+    default: 18,
+    lg: 22,
+  },
+  fontWeight: {
+    default: 'normal',
+    lg: 'bold',
+  }
+}
+
+const LinkStyled = styled(Link)`
+  font-family: Muli, sans-serif;
+  font-color: ${COLORS.blue};
+  padding-right: 40px;
+  font-size: ${(props) => TextThemes.fontSize[props.ver]}px;
+  font-weight: ${(props) => TextThemes.fontWeight[props.ver]};
+  text-decoration: none;
+
+  &:link {
+    color: ${COLORS.blue};
+  }
+  &:visited {
+    color: ${COLORS.blue};
+  }
+  &:hover {
+    color: ${COLORS.yellow};
+    text-decoration: none;
+  }
+  &:active {
+    color: ${COLORS.yellow};
+    text-decoration: none;
+  }
+`
+
+export default function NavBar(props)  {
+
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  let menuOpen = false;
+
+  const handleMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    menuOpen = false;
+    setAnchorEl(null);
+  };
+
+  let userLinks;
+  if (false) {
+    userLinks = (
+      <>
+        <LinkStyled to='/login' ver='default'>Login</LinkStyled>
+        <div/>
+        <Modal title="Sign Up" trigger={<Button theme='accent' size='smmd'> Sign Up </Button>}> <Wizard content={Wiz_content} /> </Modal>
+      </>
+    );
+  } else {
+    userLinks = (
+      <>
+        <IconButton
+          aria-label="account of current user"
+          aria-controls="menu-navbar"
+          aria-haspopup="true"
+          onClick={handleMenu}
+          color="inherit"
+          onMouseOver={handleMenu}
+        >
+          <AccountCircleIcon/>
+          <div style={{ padding:'10px' }}/>
```

Comment:
Use relative sizes instead of pixels if possible

Reply:
I think just here it's fine to be relative -- changed to relative pixel sizing everywhere else